package.json

```diff
@@ -36,10 +36,10 @@
     "root": true
   },
   "dependencies": {
-    "@enact/core": "^3.2.0",
-    "@enact/i18n": "^3.2.0",
-    "@enact/spotlight": "^3.2.0",
-    "@enact/ui": "^3.2.0",
+    "@enact/core": "^3.3.0-alpha.8",
+    "@enact/i18n": "^3.3.0-alpha.8",
+    "@enact/spotlight": "^3.3.0-alpha.8",
+    "@enact/ui": "^3.3.0-alpha.8",
```

Comment:
I don't know what is the best way to manage the dependencies. When developing enact libraries, we usually use symbolic links. So we didn't update them in our PRs. We do when releasing the libraries. But if you have any issues, it looks fine to update the version as you did.

Reply:
We really need this and we don't need to work on the other enact libraries locally.

Reply:
This will be less of an issue later on, once Enact is out of alpha. At that point, simply specifying `^3` will let you continue using the latest stable release, whatever that happens to be, and you won't get left out of important updates (scroller, hooks, new features, etc).

Right now though, since Enact is in a special period, I'd recommend checking out enact from github and using a locally linked version, since that will allow you to stay current (by occasionally pulling in the latest releases via `git pull`), even if you're not making changes to the enact repo itself. A locally linked version ignores these lines in the dependencies list here.

There's a minor additional bonus, which is that you can poke around and explore the enact codebase for examples or see how we did things a little easier.